src/components/Modal.js

```diff
@@ -190,12 +192,13 @@ const Modal = React.createClass({
           <div className='mx-modal-scrim' onClick={this.props.onRequestClose} style={Object.assign({}, styles.scrim, styles.overlay, this.props.isRelative && styles.relative)}></div>
           <div
             className='mx-modal-container'
-            ref={ref => this._modal = ref}
             style={Object.assign({}, styles.container, this.props.style)}
           >
             {this._renderTitleBar()}
             <div
+              aria-label={this.props.ariaLabel}
```

Comment:
Do you think the `aria-label` will generally be the same as the modal title? If so, I wonder if using `aria-labelledby` to point to the title would be better because the existing modals we use would not need to change.